tests/e2e/helpdesk/chats/participants/test_sync_participants.py

```
@@ -15,23 +16,29 @@ def test_list_chat_participants(chat_participants_service):
     assert all(isinstance(participant, ChatParticipant) for participant in result)
 
 
-@pytest.mark.skip(reason="Unskip after MPT-20015 completed")  # noqa: AAA01
-def test_create_chat_participant(created_chat_participant):
-    assert isinstance(created_chat_participant, ChatParticipant)
+def test_create_chat_participant(created_chat_participant, contact_id):  # noqa: AAA01
```

Ruff (0.15.10)

[warning] Invalid rule code in `# noqa`: AAA01

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)